### PR TITLE
Ability for client to send multiple messages to server

### PR DESCRIPTION
**Pull request by Himalee**

### CODE REVIEW

File: src/main/java/EchoServer.java [hunk 1]
```diff
@@ -10,7 +11,13 @@ public EchoServer(SocketManager serverSocketManager) {
     public void start(int port) throws IOException {
         serverSocketManager.connect(port);
         String clientInput = serverSocketManager.receiveString();
-        serverSocketManager.present(Message.confirmClientMessageReceived());
-        serverSocketManager.present(clientInput);
+        while(!clientInput.equals(COMMAND_TO_QUIT)) {
```
Comment (ssunday) [inline src/main/java/EchoServer.java:19]: Should there be some doc in the command explaining to hit q to quit?

Comment (Himalee) [reply src/main/java/EchoServer.java:19]: @ssunday should I document this in the README?

Comment (ssunday) [reply src/main/java/EchoServer.java:19]: That would be a good first place, but when a user launches the program how would they know? Is the expectation they'd always read the readme?

Comment (ssunday) [reply src/main/java/EchoServer.java:19]: like how would the end user know?

Help menu?

Comment (Himalee) [reply src/main/java/EchoServer.java:19]: Understood!

File: src/main/java/Message.java [hunk 2]
```diff
@@ -7,4 +7,8 @@ public static String confirmClientMessageReceived() {
     public static String serverShutsDown() {
         return "Goodbye";
     }
+
+    public static String menu() {
+        return "To quit, press 'q' and enter";
```
Comment (ssunday) [inline src/main/java/Message.java:12]: Wooh! 👏

File: src/main/java/ServerSocketManager.java [hunk 3]
```diff
@@ -1,24 +1,26 @@
...
     public String receiveString() throws IOException {
-        Scanner serverInput = new Scanner(socket.getInputStream());
-        return serverInput.nextLine();
+         return cli.getInput(socket.getInputStream());
     }
 
     public void present(String message) throws IOException {
-        PrintWriter serverOutput = new PrintWriter(new OutputStreamWriter(socket.getOutputStream()), true);
-        serverOutput.println(message);
+        cli.getOutput(socket.getOutputStream(), message);
```
Comment (amb8l) [inline src/main/java/ServerSocketManager.java]: Is this method getting output from the CLI or sending output to be displayed?

Comment (Himalee) [reply src/main/java/ServerSocketManager.java]: This method gets the output from the output stream and then displays it to the client via the CLI - is this method doing too much?

Apologies for the delayed response, coming back to this now!

**Pull request merged**